npm-recursive-audit.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```

Comment:
Why do we need this? `lerna exec npm audit fix` will do the same.

Reply:
Ah, cool, wasn't aware of that. Amended to remove the bash script in favor of just adding an npm script (and updating per the latest audit).